Framework/Kernel/src/ConfigService.cpp

```diff
@@ -465,8 +464,8 @@ void ConfigServiceImpl::configureLogging() {
     Poco::Util::LoggingConfigurator configurator;
     configurator.configure(m_pConf.get());
   } catch (std::exception &e) {
-    std::cerr << "Trouble configuring the logging framework " << e.what()
-              << '\n';
+    g_log.warning() << "Trouble configuring the logging framework " << e.what()
```

Comment:
This case has to be an exception and `cerr` is valid here the logging could not be configured so it can't be used and we have to fallback to `cerr`.